codecov.yml

```diff
@@ -1,3 +1,5 @@
+# note: `.github/codecov-upstream.yml` is basically a copy of this file, please propagate
+# changes as needed
```

Comment:
If (the concept of) `.github/codecov-upstream.yml` needs to be kept it could also just append necessary lines to this file, making this comment unnecessary.

Reply:
> append necessary lines to this file

What do you mean? Is there a way to merge the two files?

Reply:
```sh
cat >> codecov.yml <<EOF
codecov:
￼  # token from: https://codecov.io/gh/pytest-dev/pytest/settings
￼  # use same URL to regenerate it if needed
￼  token: "1eca3b1f-31a2-4fb8-a8c3-138b441b50a7"
EOF
```

Reply:
/cc @hugovk in case you want to adopt it also.

Reply:
Good idea, done. 👍 

Used a python script so I can put docs, do assertions, etc.